bindata/network/kuryr/004-daemon.yaml

```diff
@@ -24,15 +24,14 @@ spec:
       priorityClassName: system-node-critical
       initContainers:
       - name: install-cni-plugins
-        image: {{ .NodeImage }}
+        image: {{ .CNIPluginsImage }}
         command:
         - /bin/bash
         - -c
         - |
           #!/bin/bash
           set -ex
-          # Take over network functions on the node
-          cp -Rf /opt/cni/bin/* /host-cni-bin/
+          cp -f /opt/cni/bin/loopback /host-cni-bin/
```

Comment:
Judging from #237 the plugins in this particular image are in /usr/src/plugins/bin. I've built the image myself and it confirms it.

Reply:
huh. ok